qiskit_experiments/library/characterization/zz_ramsey.py

```diff
@@ -0,0 +1,339 @@
+
+        .. math::
+
+            H = h \left(\frac{f_0}{2} ZI + \frac{f_1}{2} IZ + \frac{f_{ZZ}}{4} ZZ\right)
+
+        and measures the strength :math:`f_{ZZ}` of the :math:`ZZ` term.
+        :math:`f_{ZZ}` can be described as the difference between the frequency
+        of qubit 0 when qubit 1 is excited and the frequency of qubit 0 when
+        qubit 1 is in the ground state. Because :math:`f_{ZZ}` is symmetric in
+        qubit index, it can also be expressed with the roles of 0 and 1
+        reversed.  Experimentally, we measure :math:`f_{ZZ}` by performing
+        Ramsey sequences on qubit 0 with qubit 1 in the ground state and again
+        with qubit 1 in the excited state. The standard Ramsey experiment
+        consists of putting a qubit along the :math:`X` axis of Bloch sphere,
+        waiting for some time, and then measuring the qubit project along
+        :math:`X`. By measuring the :math:`X` projection versus time the qubit
+        frequency can be inferred. See
+        :class:`~qiskit_experiments.library.characterization.T2Ramsey` and
+        :class:`~qiskit_experiments.library.characterization.RamseyXY`.
+
+        Because we are interested in the difference in qubit 0 frequency
+        between the two qubit 1 preparations rather than the absolute
+        frequencies of qubit 0 for those preparations, we modify the Ramsey
+        sequences (the circuits for the modified sequences are shown below).
+        First, we add an X gate on qubit 0 to the middle of the Ramsey delay.
+        This would have the effect of echoing out the phase accumulation of
+        qubit 0 (like a Hahn echo sequence as used in
+        :class:`~qiskit_experiments.library.characterization.T2Hahn`), but we
+        add a simultaneous X gate to qubit 1 as well.  Flipping qubit 1 inverts
+        the sign of the :math:`f_{ZZ}` term. The net result is that qubit 0
+        continues to accumulate phase proportional to :math:`f_{ZZ}` while the
+        phase due to any ZI term is canceled out. This technique allows
+        :math:`f_{ZZ}` to be measured using longer delay times than might
+        otherwise be possible with a qubit with a slow frequency drift (i.e.
+        the measurement is not sensitive to qubit frequency drift from shot to
+        shot, only to drift within a single shot).
+
+        The resulting excited state population of qubit 0 versus delay time
+        exhibits slow sinusoidal oscillations (assuming :math:`f_{ZZ}` is
+        relatively small). To help with distinguishing between qubit decay and
+        a slow oscillation, an extra Z rotation is applied before the final
+        pulse on qubit 0. The angle of this Z rotation is set proportional to
+        the delay time of the sequence so that it acts like an extra rotation
+        frequency common to the two qubit 1 preparations. By looking at the
+        difference in frequency fitted for the two cases, this common "fake"
```

Comment:
Can we say this frequency differently? Something like offset. To me "fake" sounds like an error of observed quantity (indeed it's an error from the true frequency but it's known and we can exactly control it). 

Instead of having description here, you can update documentation of [T2Ramsey](https://github.com/Qiskit/qiskit-experiments/blob/219af364f10486bfa6ed2bd57b1a1b1552c9fbae/qiskit_experiments/library/characterization/t2ramsey.py#L33-L53) and refer to it. This is a common technique but no documentation in T2Ramsey.

Reply:
In 896e548e148544f68815fcc9a1ca85cbe1d3dd84, I removed the word "fake". Maybe that is enough?

I agree that it would be good to add more description to T2Ramsey, but I don't know if it would save enough text to refer to it versus just explaining it again in the ZZRamsey documentation because it needs to be explained well enough to describe the subtraction of the 1 and 0 frequencies which doesn't apply to the T2Ramsey experiment and also to describe the `rotations` option.

Reply:
Fair enough :)